ipatests/pytest_ipa/integration/fips.py

```diff
@@ -37,6 +37,9 @@ def enable_userspace_fips(host):
     # fake Kernel FIPS mode with bind mount
     host.run_command(["mkdir", "-p", FIPS_OVERLAY_DIR])
     host.put_file_contents(FIPS_OVERLAY, "1\n")
+    host.run_command(
+        ["chcon", "-t", "sysctl_crypto_t", FIPS_OVERLAY]
+    )
```

Comment:
I think you should probably add the user too, -u system_u. Otherwise it is going to be unconfined_u I think.

Reply:
Thanks for the suggestion, PR updated.